packages/opencensus-core/src/internal/cls.ts

```diff
@@ -17,39 +17,24 @@
 // Original file from Stackdriver Trace Agent for Node.js
 // https://github.com/GoogleCloudPlatform/cloud-trace-nodejs
 
-import * as CLS from 'continuation-local-storage';
-import * as semver from 'semver';
+import * as CLS from 'cls-hooked';
 
 export type Namespace = CLS.Namespace;
-export type Func<T> = CLS.Func<T>;
-
-const useAsyncHooks: boolean = semver.satisfies(process.version, '>=8'); //&&
-// !!process.env.GCLOUD_TRACE_NEW_CONTEXT;
-
-const cls: typeof CLS = useAsyncHooks
-  ? require('./cls-ah')
-  : require('continuation-local-storage');
+// eslint-disable-next-line no-any
+export type Func<T> = (...args: any[]) => T;
```

Comment:
Not a huge fan of the `no-any` bypass here, but I just took this definition from the original `continuation-local-storage` package. Open to suggestions, though.